repos/spack_repo/builtin/packages/py_python_swiftclient/package.py

```diff
@@ -15,6 +15,7 @@ class PyPythonSwiftclient(PythonPackage):
 
     maintainers("ajkotobi")
 
+    version("4.7.0", sha256="afd7575753d8e49617adcb11550187fd0b120fcd819f1e782c0b538f2d093773")
```

Comment:
It seems that `python-swiftclient` changed their download tarball to use underscore. This will need a `url_for_version` fix as in e.g. https://github.com/spack/spack-packages/blob/8fd81a878bee41062899c10e5ad403c6c62b5b9a/repos/spack_repo/builtin/packages/py_requests_unixsocket/package.py#L29-L35

Reply:
added that.